### DISCUSSION TITLE
Literature processing and emulating a discourse graph

### DISCUSSION THREAD

**Discussion by leonardo-guedez** [BryanJenksCommunity/FAQ#159] [Ideas & Suggestions]
I’m using a separate vault for each paper I’m writing. That way it’s easier for me to keep track of the literature connections for each individual project. I’m also intentional that each project has its own ontology or MOC note.

I adapted my approach from several people who actually work with different apps:
- Roam: [Joel Chan](https://www.loom.com/share/2ec80422301c451b888b65ee1d283b40)
- Logseq: [Dario da Silva](https://youtu.be/uogIrNSYy64?t=2285) (OneStutteringMind)
- Obsidian: [Bryan Jenks](https://youtu.be/mMpVMuGBGe8?t=1929)

The idea is to create three main kinds of notes:
- research question
- claim
- evidence

I assigned a naming convention for each type. The file starts with either QUE, CLM, or EVD. I also use extra syntax for specific purposes such as to pull searches up in the quick switcher. I use two DB folder files or “databases”: one for linguistic input (QUE, CLM, EVD, etc.) and another for empirical input (the actual files for each work cited) (see screenshots).

I make sure that claim notes are linked with relevant material and have tags for either “supported by” and “opposed by”. I include this info in the frontmatter. Other types of notes are, for example, “evidence aligns ___” vs. “evidence contradicts ___”; in the DB Folder view, these things are represented by their respective markers/groups with a progressive filter, e.g., 👍🏾 claim supported by, -> <- evidence contradicts ___. I avoid using metadata that can be redundant, and instead I’m in favor of using an inline block for the reference citation in LaTeX; I also use the citekey as the actual name of the note/file or work cited (see screenshots). This is a personal preference; there are multiple ways of doing this, of course.

Bryan, I have a quick question. How can I align the names of the files to the left in the DB Folder search? They are always centered and I tried to fix it in the settings but I couldn’t. I haven’t had the time to look for custom CSS to do this either. Thanks!

Hope this info helps someone out there!

![DB Folder view 1](https://user-images.githubusercontent.com/69948069/204032486-27bdb66e-8ae9-433d-aa35-c55a03005df2.png)

![DB Folder view 2](https://user-images.githubusercontent.com/69948069/204032482-8f7c9ce9-281b-4c9a-a165-cf1f67d5eb5c.png)

![Input example](https://user-images.githubusercontent.com/69948069/204032471-c70aabd1-e1ab-4cf8-82dc-eb753219761a.png)

![Annotations and tags](https://user-images.githubusercontent.com/69948069/204032479-02b5c9f0-7e6c-49c3-a0e0-6f055dee6d23.png)

![Ontology](https://user-images.githubusercontent.com/69948069/204032474-53783739-7613-4e46-bd5c-917625970b81.png)

![Graph view](https://user-images.githubusercontent.com/69948069/204032476-94ebfc8b-9692-4aec-8933-068a4e0aa8ce.png)

**Comment by tallguyjenks** [comment]
clicking on the column name > settings > alignment
